src/integrationTest/java/uk/gov/hmcts/darts/ws/AddAudioWebServiceTest.java

```diff
@@ -161,8 +162,9 @@ void testRoutesAddAudioRequestWithAuthenticationTokenFailure(DartsGatewayClient
     @ArgumentsSource(DartsClientProvider.class)
     void testAddAudioWithAuthenticationToken(DartsGatewayClient client) throws Exception {
         authenticationStub.assertWithTokenHeader(client, () -> {
-            String soapRequestStr = TestUtils.getContentsFromFile(
-                "payloads/addAudio/register/soapRequest.xml");
+
+            // reset to make sure we do not log for the core darts operation
+            logAppender.reset();
```

Comment:
Is there not a generic place this can go rather than copying and pasting into every test? Risk would be someone else coming in would forget to add to a new test. Maybe some sort of IntegrationTest Base class that every integration test extends from?

Reply:
ah yeah this neeednt be there as  its in the integration base class

Reply:
yeah I will adjust the tests to prevent this.